design/design-doc-stacks.md

```diff
@@ -359,25 +372,41 @@ spec:
 ```yaml
 group: databases.foocompany.io
 title: Sample Controller Types
-description: |
-    This is a crd group description
+overviewShort: This is a one line plain text description of this resource group
+overview: |
```

Comment:
other `overview` fields are using `|-` but this one is using `|`, is that on purpose?

Reply:
There was no real difference.  I think I was inconsistently using `|-` in examples where I included extra new-lines to emphasis the different multiline fields, especially when the next set of fields were dense.

I've removed the new-line stripping `|-` in these examples in favor or `|`.